x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -263,34 +264,29 @@ export default ({ getService }: FtrProviderContext): void => {
         }
       });
 
-      it('400s when case is missing', async () => {
+      it('404s when case is missing', async () => {
```

Comment:
this test is testing when the case has an id but it doesn't exist. Can you add a test when the caseId parameter itself is not sent?

Reply:
This is being tested by the kibana core routing framework. I don't think there is a need for us to test that functionality.

Reply:
I didn't know that. then please ignore and just change this test name.

Reply:
Any ideas for the naming?

Reply:
maybe `404s when the case doesn't exist`